[docs] Add helper script for registryDockerCfg generation (#10042)
deckhouse/deckhouse: docs/documentation/pages/DECKHOUSE-FAQ.md

`registryDockerCfg` must be Base64-encoded.

> You can use this helper script to generate `registryDockerCfg`:

Reviewer: ```suggestion
You can use the following script to generate `registryDockerCfg`:
```
